### ISSUE TITLE
Improve FloatString and IntString classes to reduce Init time and memory allocation

### ISSUE THREAD

**Issue by Tayx94** [Tayx94/graphy#55]

**Comment by juliolitwin** [comment]
Yo @Tayx94, ZString can be a possible solution?

Link: https://github.com/Cysharp/ZString/

**Comment by Tayx94** [comment]
Hey @juliolitwin , sorry for the delay. It's not a concatenation issue, it's just a caching thing, because in order to avoid memory allocation we create the strings and store them in memory. I reduced it a bit for this release, but maybe in the future we can move to TMPro directly

**Comment by nikescar1** [comment]
Yeah, unfortunately that is the case. I wouldn't think it would matter too much with something like Graphy since all values have a set position and known max number of digits. It certainly isn't flexible but that is a fair trade for the performance in the places I'm using it (lap timing, rpm, etc)
